routes/operations.js

```diff
@@ -0,0 +1,16 @@
+import { Router } from "express";
+const router = Router();
+import { operations } from "cyberchef/src/node/index.mjs";
+
+/**
+ * operationsGet
+ */
+router.get("/", async (req, res, next) => {
```

Comment:
a bit of a nit: please could you make this function a `function` rather than an arrow function, just for consistency please.